ibm_catalog.json

```diff
@@ -101,6 +106,242 @@
+                "options": [
+                  {
+                    "displayname": "lite",
+                    "value": "lite"
+                  },
+                  {
+                    "displayname": "7-day",
+                    "value": "7-day"
+                  },
+                  {
+                    "displayname": "14-day",
+                    "value": "14-day"
+                  },
+                  {
+                    "displayname": "30-day",
+                    "value": "30-day"
+                  },
+                  {
+                    "displayname": "hipaa-30-day",
+                    "value": "hipaa-30-day"
+                  }
+                ]
+              },
+              {
+                "key": "log_analysis_service_endpoints",
+                "options": [
+                  {
+                    "displayname": "public",
+                    "value": "public"
+                  },
+                  {
+                    "displayname": "private",
+                    "value": "private"
+                  },
+                  {
+                    "displayname": "public-and-private",
+                    "value": "public-and-private"
+                  }
+                ]
+              },
+              {
+                "key": "log_analysis_tags"
+              },
+              {
+                "key": "enable_log_archive"
+              },
+              {
+                "key": "log_archive_api_key"
+              },
+              {
+                "key": "enable_platform_logs",
+                "default_value": "false"
+              },
+              {
+                "key": "cloud_monitoring_instance_name"
+              },
+              {
+                "key": "cloud_monitoring_plan",
+                "options": [
+                  {
+                    "displayname": "lite",
+                    "value": "lite"
+                  },
+                  {
+                    "displayname": "graduated-tier",
+                    "value": "graduated-tier"
+                  }
+                ]
+              },
+              {
+                "key": "cloud_monitoring_service_endpoints",
+                "options": [
+                  {
+                    "displayname": "public",
+                    "value": "public"
+                  },
+                  {
+                    "displayname": "private",
+                    "value": "private"
+                  },
+                  {
+                    "displayname": "public-and-private",
+                    "value": "public-and-private"
+                  }
+                ]
+              },
+              {
+                "key": "cloud_monitoring_tags"
+              },
+              {
+                "key": "enable_platform_metrics",
+                "default_value": "false"
+              },
+              {
+                "key": "add_bucket_name_suffix"
+              },
```

Comment:
this one ..you can move after cos instance details. Its like, Cos instance and then bucket related details.